src/Covid19Dashboard/standalone/package.json

```diff
@@ -54,7 +54,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --mode development",
+    "start": "set \"HOST=local-covid19.planx-pla.net\" && webpack-dev-server --mode development",
```

Comment:
why change from default?

Reply:
This is specifically related to fetching data from the OCC s3 bucket for this project. The bucket is locked down to specific urls and and adding localhost to the CORS rules is not allowed so a specific url is needed for testing the standalone.

Reply:
if needed can take this out but this should only affect the standalone dev environment